tests/tests_update.yml

```diff
@@ -7,14 +7,26 @@
       vars:
         crypto_policies_policy: LEGACY
         crypto_policies_reload: false
-
```

Comment:
I was keeping spaces between separate tasks to make the yaml a bit more readable.